src/pages/CustomLessonsDisplay/CustomLessonsDisplay.js

```diff
@@ -56,6 +57,10 @@ function CustomLessonsDisplay(props) {
     setDisplayLessonInfo(display);
   }
 
+  function handleDeletedLesson() {
+    setDeletedLesson(!deletedLesson);
+  }
```

Comment:
was this the fix for the lessons not updating problem?

Reply:
Yup! So this signals to the LessonsDisplay to recall the firebase endpoint and update the lessons displayed on the screen in useEffect()

Reply:
could you explain this? is the value of deletedlesson ignored and it's only used to trigger `useEffect()`?

Reply:
wdym the value of deletedLesson ignored?

Yeah your interpretation is right, I made this specifically just to trigger useEffect(), to signal to the parent component displaying the customLessons (customLessonsDisplay) to update the lessons. Since we're not redirecting back to the same page, I think this is the best way to do it but correct me if I'm wrong if you have a better implementation!

Reply:
i just meant that it doesnt matter if it's true or false because its changing updates the component
i think it should be fine, it's only a little janky